docs_src/microservices/application/AppFunctionContextAPI.md

```diff
@@ -61,30 +71,46 @@ Returns a `LoggingClient` to leverage logging libraries/service utilized through
 
+### CommandClient
 
-### CommandClient() 
+`CommandClient() interfaces.CommandClient`
 
 Returns a `CommandClient`  to leverage Core Command's `Command` API. See [interface definition](https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/interfaces/command.go) for more details. Useful for sending commands to devices. Note if Core Command is not specified in the Clients configuration, this will return nil.
 
+### NotificationClient
 
-### NotificationClient()
+`NotificationClient() interfaces.NotificationClient`
 
 Returns a `NotificationClient` to leverage Support Notifications' `Notifications` API. See [interface definition](https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/interfaces/notification.go) for more details. Useful for sending notifications. Note if Support Notifications is not specified in the Clients configuration, this will return nil.
 
-### SubscriptionClient()
+### SubscriptionClient
+
+`SubscriptionClient() interfaces.SubscriptionClient`
+
 Returns a `SubscriptionClient` to leverage Support Notifications' `Subscription` API. See [interface definition](https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/interfaces/subscription.go) for more details. Useful for creating notification subscriptions. Note if Support Notifications is not specified in the Clients configuration, this will return nil.
 
-### DeviceServiceClient()
+### DeviceServiceClient
+
+`DeviceServiceClient() interfaces.DeviceServiceClient`
+
 Returns a `DeviceServiceClient` to leverage Core Metadata's `DeviceService` API. See [interface definition](https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/interfaces/deviceservice.go) for more details. Useful for querying information about Device Services. Note if Core Metadata is not specified in the Clients configuration, this will return nil. 
 
-### DeviceProfileClient()
+### DeviceProfileClient
+
+`DeviceProfileClient() interfaces.DeviceProfileClient`
+
 Returns a `DeviceProfileClient` to leverage Core Metadata's `DeviceProfile` API. See [interface definition](https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/interfaces/deviceprofile.go) for more details. Useful for querying information about Device Profiles and is used by the `GetDeviceResource` helper function below. Note if Core Metadata is not specified in the Clients configuration, this will return nil. 
 
-### DeviceClient()
+### DeviceClient
+
+`DeviceClient() interfaces.DeviceClient`
+
 Returns a `DeviceClient` to leverage Core Metadata's `Device` API. See [interface definition](https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/interfaces/device.go) for more details. Useful for querying information about Devices. Note if Core Metadata is not specified in the Clients configuration, this will return nil. 
 
 ### Note about Clients
```

Comment:
This wasn't one of the fixes in the issue, but... the example below need to be in a `!!! example box